Description: verbosly die if /proc is not mounted
 Let patch-wrapper die with an informative message when /proc is not mounted
 althrough it needs it to pass the patching request to GNU patch since it
 fails to understand all options.
Forwarded: submitted 2012-12-19

--- a/bin/patch-wrapper.in
+++ b/bin/patch-wrapper.in
@@ -171,6 +171,10 @@ then
     if [ -n "$opt_input" ]
     then
 	patch=$opt_input
+    elif [ ! -e /proc/self ]
+    then
+        echo "patch-wrapper: /proc not mounted!" >&2
+	exit 1
     elif [ -e /proc/self/fd/0 ]
     then
 	patch=$(readlink /proc/self/fd/0)
